### DISCUSSION TITLE
Allow keeping multiplayer leaderboard expanded in-game

### DISCUSSION THREAD

**Discussion by Highmagik** [ppy/osu#22040] [Q&A] [answered]
Is there a way to always have scores shown during the game without pressing ctrl button? Not just places

**Comment by peppy** [comment]
There's not right now, but this will probably be added as a setting.

**Reply by natsukagami** [reply]
With #32939 implemented, perhaps this can be easily exposed as a skinnable config? I saw we already have a prop for that (`forceExpand`) in the skinnable.

**Comment by peppy** [comment, marked as answer]
https://github.com/ppy/osu/pull/33630
